admin/vagrant-acceptance-targets/centos-7/Vagrantfile

```diff
@@ -10,6 +10,30 @@ VAGRANTFILE_API_VERSION = "2"
 
 ENV['VAGRANT_DEFAULT_PROVIDER'] = 'virtualbox'
 
+$bootstrap_control = <<EOF
```

Comment:
So why is this logic both here and in the tutorial `Vagrantfile.template`? We want acceptance tests to be using something as close as possible to tutorial Vagrant, for realism, so having two copies of setup code is bad in that sense. It's also two copies of code, which is just bad in general.

How about have `run-acceptance-tests` just make a rendered copy of the tutorial's Vagrantfile template?